scai-gen/go.mod

```diff
@@ -1,18 +1,19 @@
-module scai-gen
+module github.com/in-toto/scai-demos/scai-gen
```

Comment:
I think my only outstanding thought is re the dir structure. Typically go.mod ought to go in the root of the repo. That's also what you'd expect for main.go. This is because of the non-go packages that are also here, right? Either way I'll defer to @pxp928.

Reply:
Right, yes, the current structure is because of the non-Go packages. Happy to move things around if others have stronger feelings about this.